tests/serverpod_test_server/test_e2e/method_streaming/open_stream_test.dart

```diff
@@ -0,0 +1,21 @@
+@Timeout(Duration(minutes: 5))
```

Comment:
Question: How come this needs a 5 minute timeout?

Reply:
This is not required, will remove it!